CSharp_Net_module1_1_4_lab/CSharp_Net_module1_1_4_lab/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Net_module1_1_4_lab
+{
+    class Program
+    {
+        // 1) declare enum ComputerType
+        enum ComputerType
+        {
+            Desktop,
+            Laptop,
+            Server
+        }
+
+        // 2) declare struct Computer
+        struct Computer
+        {
+            public ComputerType Type;
+            public byte Cores;
+            public float HGz;
+            public byte Memory;
+            public int HDD;
+
+            public override string ToString()
+            {
+                return this.Type.ToString();
+            }
+        }
+
+        static void Main(string[] args)
+        {
+            var desktop = new Computer();
+            var laptop = new Computer();
+            var server = new Computer();
+
+            desktop.Type = ComputerType.Desktop;
+            desktop.Cores = 4;
+            desktop.HGz = 2.5F;
+            desktop.Memory = 6;
+            desktop.HDD = 500;
+
+            laptop.Type = ComputerType.Laptop;
+            laptop.Cores = 2;
+            laptop.HGz = 1.7F;
+            laptop.Memory = 4;
+            laptop.HDD = 250;
+
+            server.Type = ComputerType.Server;
+            server.Cores = 8;
+            server.HGz = 3F;
+            server.Memory = 16;
+            server.HDD = 2048;
+
+            Computer[][] departments = new Computer[4][] {
+            new Computer[] {desktop, desktop, laptop, laptop, server},
+            new Computer[] {laptop, laptop, laptop},
+            new Computer[] {desktop, desktop, desktop, laptop, laptop},
+            new Computer[] {desktop, laptop, server, server}
+            };
+
+            // preparing variables
```

Comment:
you don't need to leave such comments

Reply:
Fixed in commit f5e31f3a4612e2e146c57d592bbe6505c3d1dea4